### DISCUSSION TITLE
MSc%40UofT/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#194] [Announcements]
# MSc%40UofT/

None

https://opencs.app/MSc%40UofT/

**Comment by ThisGuyIsNotAJumpingBear** [comment]
坐标UofT，MSc是full funding thesis master，相当于博士预备班，然后政府/大学是不给国际生批经费的（往年有一回放开过，但是后面又收回去了）。所以想到多伦多读master就去MScAc，搞研究就直接申请phd。没身份就无视这个选项就好。

Quote by 我的导师：“You can have a try if you want to apply, but in most of time we just take your application fee and that's it.”

**Comment by YuRuiii** [comment]
https://web.cs.toronto.edu/graduate/msc

International Applicants
International applicants considering applying to graduate programs in Computer Science are strongly encouraged to apply to either the Master of Science in Applied Computing or the Direct-Entry PhD Program. 

International applicants are not considered for the MSc in Computer Science.

:(
